storage_scrubber/src/pageserver_physical_gc.rs

```diff
@@ -101,23 +172,70 @@ async fn maybe_delete_index(
                     Ok(e) => e,
                     Err(_) => {
                         tracing::warn!("Bad last_modified time: {last_modified:?}");
-                        return;
+                        return false;
                     }
                 }
             }
         },
         Err(e) => {
             tracing::warn!("Failed to HEAD {key}: {e}");
             summary.remote_storage_errors += 1;
-            return;
+            return false;
         }
     };
-    if &age < min_age {
+    let old_enough = &age > min_age;
+
+    if !old_enough {
         tracing::info!(
             "Skipping young object {} < {}",
             age.as_secs_f64(),
             min_age.as_secs_f64()
```

Comment:
I think sprinkling in some `humantime::Duration` would be appreciated by whatever poor soul will have to read these logs?

Reply:
Pourquoi non.  6be4b75ce